packages/reason-relay/__tests__/Test_pagination-tests.js

```diff
@@ -10,115 +10,122 @@ describe("Pagination", () => {
     queryMock.mockQuery({
       name: "TestPaginationQuery",
       variables: {
-        groupId: "123"
+        userId: "123"
```

Comment:
Like we talked about on Discord, but the changes to this test should probably be its own test rather than change the existing one.

Reply:
yes definitely, I'll change that, both tests are valuable, should I rename them to `Test_paginationUnion.re` for the original one and `Test_paginationRefetch.re` for this one so the filenames are more explicit?

Reply:
Yeah, do that, that'll help!